JobSchema.md

```diff
@@ -0,0 +1,221 @@
+Job Schema Service
+==================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+## Summary
+
+Job Schema Service (JSS) takes a request containing data for a job posting and returns a response conforming to the [schema.org](﻿http://schema.org/) [JobPosting schema](﻿http://schema.org/JobPosting).
+
+## Request Structure
+```
+{
+  "job_data": {
+    "job_display_url": string,
+    "job_title": string,
+    "description": string,
+    "requirements": string,
+    "start_date": string,
+    "end_date": string,
+    "employee_type": string,
+    "company_data": {
+      "image_url": string,
+      "company_url": string,
+      "company_did": string,
+      "company_name": string
+    },
+    "industries": [
+      string
+    ],
+    "location": {
+      "city": string,
+      "state": string,
+      "street_address": string,
+      "postal_code": string,
+      "country_code": string,
+      "latitude": string,
+      "longitude": string
+    },
+    "required_education": string,
+    "min_experience": number,
+    "max_experience": number,
+    "pay_info": {
+      "currency_code": string,
+      "pay_low": number,
+      "pay_high": number,
+      "pay_period": string,
+      "pay_bonus": number,
+      "pay_commission": number
+    },
+    "onet": {
+      "code": string,
+      "title": string
+    },
+    "display_job_id": string,
+    "job_did": string,
+    "talent_network_did": string
+  },
+  "jpan_data": {
+    "experience_level": {
+      "max_years": number,
+      "min_years": number
+    },
+    "education_level": {
+      "description": string
+    },
+    "working_hours": string,
+    "extracted_text": {
+      "job": {
+        "text": string
+      },
+      "candidate": {
+        "text": string
+      },
+      "conditions": {
+        "text": string
+      }
+    },
+    "skills": {
+      "4.0": [
+        {
+          "normalized_term": string
+        },
+      ]
+    }
+  },
+  "company_normalization_data": {
+    "master_company": {
+      "naics_description": string
+    }
+  },
+  "salary_data": {
+    "regional_estimate": {
+      "percentiles": [
+        {
+          "percentile": 10,
+          "annual_salary": number,
+          "hourly_salary": number
+        },
+        {
+          "percentile": 50,
```

Comment:
are we not calling this "median" anymore

Reply:
The JobPosting schema (and therefore the response) calls it median but the request does not. In general, I've flattened the request objects that come from other services (this one is from emsi) but haven't changed the field names.